### DISCUSSION TITLE
런칭페스티벌 준비

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#480] [이야기해봐요]
## 시연
prod 서버로 시연

### 시나리오
+ 서비스 소개
  + 공감을 이끌어 낼 수 있는 내용들을 소개하기
+ 
+

**Comment by kyum-q** [comment]
## 시연 기능

- 관리 기능 (내 템플릿)
- 탐색 기능 (둘러보기)
- 생성 / 삭제

**Comment by jminkkk** [comment]
## 시연 - 테스트 데이터 추가
- 시연이 충분히 가능한 테스트 데이터를 추가한다.
- 언제까지? 얼마나?

**Comment by kyum-q** [comment]
## 부스 운영

- 스티커 제작
   - 디자인 : 치타 100, 로고 100

**Comment by jminkkk** [comment]
### 서비스 소개
안녕하세요 와주셔서 감사합니다.

**Reply by kyum-q** [reply]
여러분, 이런 경험 한 번쯤 있으시죠? 🤔

"아, 그때 그 코드 어디에 썼더라..."

이전 프로젝트 파일을 뒤적거리며 찾아 헤매던 그 순간들 말이에요.

그럴 때마다 '내가 작성한 코드를 쉽게 찾을 수 있는 방법 없을까?' 하고 고민하셨다면, 여기 「코드잽⚡️」 이 있습니다!


코드잽은 이런 분들에게 딱이에요 🙌
자주 쓰는 코드 템플릿을 간편하게 저장하고 싶은 분
프로젝트 파일을 뒤적거리는 대신 필요한 코드를 빠르게 찾고 싶은 분
체계적으로 코드를 정리하고 싶지만 방법을 모르셨던 분

어떻게 사용하냐고요? 아주 간단해요 🔥
유용한 코드를 발견하면 바로 저장!
직관적인 분류 시스템으로 깔끔하게 정리!
필요할 때 빠르게 검색해서 사용!

상상해 보세요!
JWT와 리프레시 토큰으로 로그인 기능을 구현해야 하는 순간, 코드잽을 한 번 클릭하니 완벽한 코드 템플릿이 눈앞에 펼쳐집니다.


저희 또한 개발자로서, 실제 개발자들의 고민을 바탕으로 이 서비스를 만들었어요.

그 경험으로 탄생한 코드잽은 개발자의, 개발자에 의한, 개발자를 위한 서비스!
복잡한 건 NO! 효율적인 분류 시스템과 누구나 10초만에 마스터할 수 있는 UI로 여러분의 코딩 라이프를 업그레이드해 드립니다.


"코드 찾느라 고생했던 날은 이제 안녕!"

코드잽과 함께라면, 개발 속도는 번개처럼 빨라질 겁니다. 코드를 저장하고 검색하는 과정이 더 이상 지루하고 번거롭지 않아요.

코드잽, 여러분의 개발 인생에 zap⚡️! 당신의 손끝에서 코드가 춤춥니다! 🕺🏻💃🏻
